openms_python/py_aasequence.py

```
@@ -277,4 +391,37 @@ def has_suffix(self, suffix: str) -> bool:
+        if not modified:
+            return self.unmodified_sequence
+        
+        else:
+            if mod_format == 'default':
+                return self._sequence.toString()
+            elif mod_format == 'unimod':
+                return self._sequence.toUniModString()
+            elif mod_format == 'bracket':
+                return self._sequence.toBracketString()
+            else:
+                raise ValueError(f"Unsupported mod_format: {mod_format}, supported are 'unimod' and 'bracket'")
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)